### ISSUE TITLE
it seems to delete all volumes, not just orphaned

### ISSUE THREAD

**Issue by vitaly** [chadoe/docker-cleanup-volumes#11]
just one volume directory:

```
root@boot2docker:/mnt/sda1/var/lib/docker/volumes# ls -la
total 12
drwx------    3 root     root          4096 Jun 24 21:21 .
drwxr-xr-x    9 root     root          4096 Jun 24 21:24 ..
drwxr-xr-x    3 root     root          4096 Jun 24 21:21 bd1e586a629637ffb1153847260e63f810c43d7f9f28f382eecacf9f4a5eb749
```

and as you can see it IS used by a running container:

```
✗ docker inspect --format '{{.Volumes}}' database                                                                                                                                                                                                                                                               ruby-2.2.0 (master) deedf83
map[/var/lib/postgresql/data:/mnt/sda1/var/lib/docker/volumes/bd1e586a629637ffb1153847260e63f810c43d7f9f28f382eecacf9f4a5eb749/_data]
```

now running the script:

```
✗ docker run -v /var/run/docker.sock:/var/run/docker.sock -v /var/lib/docker:/var/lib/docker --rm martin/docker-cleanup-volumes --dry-run                                                                                                                                                                       ruby-2.2.0 (master) deedf83

Delete unused volume directories from /var/lib/docker/volumes
Would have deleted bd1e586a629637ffb1153847260e63f810c43d7f9f28f382eecacf9f4a5eb749

Directory /var/lib/docker/vfs/dir does not exist, skipping.
```

**Comment by chadoe** [comment]
Hi, which version of docker are you running on your host?

**Comment by chadoe** [comment]
Should be fixed with 01cbf4b3b2888be094f00aacaf6f36118c05ddf8

```
docker pull martin/docker-cleanup-volumes
```

to update to latest image.

Thanks for reporting.
